drivers/bus/isapnp/isapnp.c

```diff
@@ -848,12 +692,70 @@ IsaPnpCreateLogicalDeviceResources(
 
     ResourceList->Count = 1;
     ResourceList->List[0].InterfaceType = Isa;
-    ResourceList->List[0].PartialResourceList.Version = 1;
-    ResourceList->List[0].PartialResourceList.Revision = 1;
+    ResourceList->List[0].PartialResourceList.Version = 0;
+    ResourceList->List[0].PartialResourceList.Revision = 0x3000;
```

Comment:
Could we have a define const for this one? (in the header file or somewhere)

Reply:
Reverted it back to the ver.1 rev.1. I don't see this worth changing.